File: Service/Billing/RMA.Service.Billing.Contracts/Entities/PolicyProductCategory.cs
using RMA.Service.ClientCare.Contracts.Entities.Product;

using System.Collections.Generic;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class PolicyProductCategory
    {
        public int PolicyId { get; set; }
        public ProductOption ProductOption { get; set; }
        public string PolicyNumber { get; set; }
        public string ProductCategory { get; set; }
        public string ProductDescription { get; set; }
        public List<PolicyProductCategory> CategoryPolicies { get; set; }
        public int RmaBankAccountId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/policy-product-category.ts
import { ComplianceResult } from "projects/clientcare/src/app/policy-manager/shared/entities/compliance-result";
import { ProductOption } from "projects/clientcare/src/app/product-manager/models/product-option";

export class PolicyProductCategory{
    policyId: number;
    productOption: ProductOption;
    policyNumber: string;
    productCategory: string;
    productDescription: string;
    categoryPolicies: PolicyProductCategory[];
    productBalance: number;
    complianceResult: ComplianceResult;
    roleplayerId: number;
    rmaBankAccountId: number;
}